hyperchains.tex

```diff
@@ -1,33 +1,32 @@
 \section{Hyperchains design}
 \graphicspath{ {./images/} }
 
-The previous approaches had a lot to offer, but considering their weaknesses it is worth
-thinking of some alternative solutions. PoW seems to work well only with big
-computational effort being burned and PoS suffers from a huge amount of security
-holes. The complicated algorithms required for plugging them usually
-either do not solve
-the problem at all, moving it further to another layer of abstraction or introducing
-extreme complexity on the protocol level.
+The previous approaches had a lot to offer, but considering their weaknesses it
+is worth thinking of some alternative solutions. PoW seems to work well only
+with big computational effort being burned and PoS suffers from a huge amount of
+security holes. The complicated algorithms required for plugging them usually
+either do not solve the problem at all, moving it further to another layer of
+abstraction or introducing extreme complexity on the protocol level.
 
 Here we present a hybrid strategy that benefits from the stability of PoW
-solutions but offers the scalability of PoS systems. A Hyperchain is a
-special kind of blockchain that sticks to an already existing chain. They are
-going to be called child and parent chain, respectively\cite{hyperchains}.
+solutions but offers the scalability of PoS systems. A Hyperchain is a special
```

Comment:
Strictly speaking, it utilizes a presumably more secure parent chain to achieve higher security for itself. The parent chain doesn't need to be PoW per se. The child chain doesn't care _how_ the parent chain achieves its security. This is also basically noted below.